packages/components/src/DataList/DataList.css

```diff
@@ -35,6 +35,35 @@
   border-bottom: var(--border-base) solid var(--color-border);
 }
 
+.selectable {
+  display: grid;
+  column-gap: var(--space-small);
+  align-items: flex-start;
+  grid-template-columns: max-content minmax(0px, auto);
+
+  @media (--medium-screens-and-up) {
+    align-items: center;
+  }
+}
+
+.listItem .selectable > :first-child {
+  padding-top: var(--space-smaller);
+  opacity: 1;
+  transition: opacity var(--transition-properties);
+  --transition-properties: var(--timing-quick) ease-in-out;
```

Comment:
do you intend to reuse this?

Reply:
At the moment no. However we _may_ want it in the future. We can remove it if it isn't needed during select all